firmware/application/apps/ui_mictx.cpp

```diff
@@ -362,42 +362,44 @@ MicTXView::MicTXView(
 	rx_lna = receiver_model.lna();
 	field_rxlna.on_change = [this](int32_t v) {
 		rx_lna = v;
-		receiver_model.set_lna(v);
+		if(rx_enabled)
```

Comment:
hi! Wouldn't it be better to replace all these if (rx_enabled) and just inject lna, vga, rx_amp in the rxaudio() function, while enabling RX ?  (as we did with those parameters for TX, but inside set_tx() ?

Reply:
This enables dynamic tuning when RX enabled, which is more convenience.
These parameters will be set when enabling RX, just like set_tx().

Reply:
yes you are right! by placing the lna, vga and frequency right where it changes, if rx_enabled, then you will be able to listen to the changes right away!